docker/README.md

```diff
@@ -1,14 +1,15 @@
 ### Building Redox using Docker images with the toolchain
 
-*All you need is git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
+*All you need is rustup, git, make, qemu, fuse and docker. The method requires a non-privileged user able to run the `docker` command, which is usually achieved by adding the user to the `docker` group.*
```

Comment:
Sorry, I removed my previous comment because I was facing an issue testing this and I didn't want to clutter the thread.

The issue was the following, when I ran the container with `make pull && make all` or just `make update && make all`, I ended up with `xargo` not being found any more. It seems like your suggestion to `make pull all` instead does the trick.

However, syncing the submodules from the container doesn't seem to be a good idea. I found the sync was very long. In the network monitor, I noticed as much data being uploaded as dowloaded. My guess is that the communication between the host and the container uses the network stack. So I guess we are better off syncing from the host, as you proposed before.

So my last proposal is to use the new step 2 you proposed (already commited) and change the run command to `make update all`. This way, we won't execute `make clean` every time and will keep the `README` simple.